docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-with-monitoring-and-telemetry.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Modernize your apps with monitoring and telemetry
 description: Modernize existing .NET applications with Azure Cloud and Windows containers | Modernize your apps with monitoring and telemetry
-ms.date: 04/30/2018
+ms.date: 12/21/2020
 ---
 # Modernize your apps with monitoring and telemetry
 
 When you run an application in production, it's critical that you have insights about how your application is performing. Is it performing at a high level? Are users getting errors, or is the application stable and reliable? You need rich performance monitoring, powerful alerting, and dashboards to help ensure that your application is available and performing as expected. You also need to be able to see quickly if there's a problem, determine how many customers are affected, and perform a root-cause analysis to find and fix the issue.
 
 ## Monitor your application with Application Insights
 
-Application Insights is an extensible Application Performance Management (APM) service for web developers who work on multiple platforms. Use it to monitor your live web application. Application Insights automatically detects performance anomalies. It includes powerful analytics tools to help you diagnose issues, and to help you understand what users actually do with your app. Application Insights is designed to help you continuously improve performance and usability. It works for apps on a wide variety of platforms, including .NET, Node.js, and J2EE, whether hosted on-premises or in the cloud. Application Insights integrates with your DevOps processes, and has connection points to a variety of development tools.
+Application Insights is an extensible Application Performance Management (APM) service for web developers who work on multiple platforms. Use it to monitor your live web application. `Application Insights` automatically detects performance anomalies. It includes powerful analytics tools to help you diagnose issues, and to help you understand what users actually do with your app. Application Insights is designed to help you continuously improve performance and usability. It works for apps on a wide variety of platforms, including .NET, Node.js, and J2EE, whether hosted on-premises or in the cloud. Application Insights integrates with your DevOps processes, and has connection points to various development tools.
 
-Figure 4-10 shows an example of how Application Insights monitors your application and how it surfaces those insights to a dashboard.
+Figure 4-10 shows an example of how `Application Insights` monitors your application and how it surfaces those insights to a dashboard.
```

Comment:
```suggestion
Figure 4-10 shows an example of how Application Insights monitors your application and how it surfaces those insights to a dashboard.
```

Same comment as above, if you're looking to emphasize this proper brand name you could either bold it, italicize it, or both - but do so sparingly.